File: beecrowd/beginner/c/1117.c
#include <stdio.h>

int main() {
    float score, total_score = 0;
    int count = 0;

    while (1) {
        scanf("%f", &score);

        if (score >= 0 && score <= 10) {
            total_score += score;
            count += 1;
        } else {
            printf("nota invalida\n");
        }

        if (count == 2) break;
    }

    printf("media = %.2f\n", total_score / 2);
    
    return 0;
}

File: beecrowd/beginner/python/1117.py
sum = count = 0

while 1:
    n = float(input())

    if 0 <= n <= 10:
        sum += n
        count += 1
    else:
        print("nota invalida")

    if count == 2:
        break

print(f"media = {sum / 2:.2f}")
